code/game/objects/machinery/overwatch.dm

```diff
@@ -649,56 +680,37 @@ GLOBAL_LIST_EMPTY(active_cas_targets)
 						wanted_marine.sec_hud_set_security_status()
 					return
 
-/obj/machinery/computer/camera_advanced/overwatch/proc/transfer_squad()
+/obj/machinery/computer/camera_advanced/overwatch/proc/transfer_squad(mob/living/carbon/human/transfer_marine, datum/squad/new_squad)
 	if(!usr || usr != operator)
 		return
-	if(!current_squad)
-		to_chat(usr, "[icon2html(src, usr)] [span_warning("No squad selected!")]")
-		return
-	var/datum/squad/S = current_squad
-	var/mob/living/carbon/human/transfer_marine = tgui_input_list(usr, "Choose marine to transfer", null, current_squad.get_all_members())
-	if(!transfer_marine)
-		return
 
 	if(!transfer_marine.job)
 		CRASH("[transfer_marine] selected for transfer without a job.")
 
-	if(S != current_squad)
-		return //don't change overwatched squad, idiot.
-
-	if(!istype(transfer_marine) || transfer_marine.stat == DEAD) //gibbed, decapitated, dead
+	if(!istype(transfer_marine)) //gibbed
```

Comment:
Skipping through removed a lot of checks, 
???

Reply:
The checks were literally just restrictive, I double checked and can't find anything that would go wrong by removing them. As far as I can tell they're only CM relics with 0 actual purpose